packages/core/src/routes/swagger.ts

```diff
@@ -17,22 +19,52 @@ type MethodMap = {
   [key in OpenAPIV3.HttpMethods]?: OpenAPIV3.OperationObject;
 };
 
+// Parameter serialization: https://swagger.io/docs/specification/serialization
+const buildParameters = (
+  parameters: unknown,
```

Comment:
nit: maybe zodParamters could be more clear.

Reply:
updated.